packages/e2e-tests/tests/object_preview-01.test.ts

```diff
@@ -43,7 +43,7 @@ test(`object_preview-01: expressions in the console after time warping`, async (
   await executeTerminalExpression(page, "Error('helo')");
   await verifyConsoleMessage(page, "Error: helo");
 
-  await executeTerminalExpression(page, 'function f() { throw Error("there"); }()');
+  await executeTerminalExpression(page, '(function f() { throw Error("there"); })()');
```

Comment:
This was being made possible by the stuff removed in https://github.com/replayio/devtools/commit/d45fcbb710c0c152d4e73d5f74209e2eedc24b72 so broke when that landed.